lib/index.js

```diff
@@ -216,6 +218,30 @@ const logsHandler = async function logsHandler(options) {
   await logs(options.function, options.tail, options.since, logStream);
 };
 
+/**
+ * Retrieve account usage stats
+ *
+ * @param {object} options - Command line options.
+ */
+const statsHandler = async function statsHandler(options) {
+  const metrics = await stats(options.since, options.until);
+  // eslint-disable-next-line no-param-reassign
+  metrics.metrics.forEach((m) => { delete m.account; });
```

Comment:
If we bring `lodash`, we can use `omit` to avoid `delete`.

metrics.metrics = metrics.metrics.map((m) => omit(m, 'account'));

Reply:
As a convention, `delete` implies some mutable state, e.g. using an object as a hash, and here we are discarding the metrics as soon as they are printed.

In this case it's simple to understand the intention from the code, it may only be confusing in the future.